data-table/__tests__/table.js

```diff
@@ -4,6 +4,113 @@ beforeEach(() => {
     document.body.innerHTML = '<div id="div-id"></div>'
 });
 
+describe('basic tests to ensure createDataTable can function well', () => {
```

Comment:
Remember that this pull request was about index.md - don't need to have all this extra test stuff added. Not the end of the world since #23 should be merged into `main` soon but, you know, just for future reference.